salt/elife-bot/processes.sls

```diff
@@ -33,6 +33,7 @@ elife-bot-processes-start:
             - elife-bot-processes-task
         - watch:
```

Comment:
It doesn't make sense for me to use `watch` or `listen` in a `cmd.run` state because it is always executed no matter the conditions. An `onlyif` could work, but this should already be run on every build.